app/service/contact_svc.py

```diff
@@ -110,8 +110,8 @@ async def _save(self, result):
                         loop.create_task(self.get_service('learning_svc').learn(operation.all_facts(), link, result.output))
             else:
                 self.get_service('file_svc').write_result_file(result.id, result.output)
-        except Exception as e:
-            self.log.debug('save_results exception: %s' % e)
+        except Exception:
+            self.log.exception('Unexpected error occurred while saving link')
```

Comment:
Making this more visible as it wasn't clear why tests/operations were failing unless running with debug logging enabled (even then we lost the stack trace).